client/src/modules/payroll/rubrics/templates/ipr.tmpl.html

```diff
@@ -0,0 +1,5 @@
+<div class="ui-grid-cell-contents text-left">
+  <div ng-if="row.entity.is_ipr">
+  	<i class="fa fa-check-square"></i>
```

Comment:
In my experience at HEV, users don't really know what the icons are.  Plus they have resizing issues in the grid (many users with poor eyesight increase the screen size and the icons then look very distorted).  It would be better to use "oui/non" or "true/false" for boolean values, I think.